platform/mellanox/mft/Makefile

```diff
@@ -35,7 +35,7 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	wget -O $(MFT_TGZ) http://www.mellanox.com/downloads/MFT/$(MFT_TGZ)
 	tar xzf $(MFT_TGZ)
 
-ifeq ($(BLDENV), bullseye)
+ifneq (,$(findstring bullseye,$(DEST)))
```

Comment:
Could you explain this change?

Reply:
Yes, this checks if the target is bullseye or buster by looking at the destination path for the target.

Reply:
your code is only check bullseye? How it is different with original code?

Reply:
BLDENV is not exported to Makefiles, thus this was not functioning.

Reply:
However I notice is largely used by many other makefile. Is it something missing in build infra?